README.md

```diff
@@ -1113,6 +1113,32 @@ _You can enable the following settings in Xcode by running [this script](resourc
 
     </details>
 
+* <a id='redundant-get'></a>(<a href='#redundant-get'>link</a>) **Omit the `get` clause from a computed variable declaration that doesn't also have a `set`, `willSet`, or `didSet` clause.** [![SwiftFormat: redundantGet](https://img.shields.io/badge/SwiftFormat-redundantGet-7B0051.svg)](https://github.com/nicklockwood/SwiftFormat/blob/master/Rules.md#redundantGet)
```

Comment:
I see that we added this rule to the Style section under the Functions subsection. Do you think the Patterns section is a more appropriate home?

Reply:
Makes sense, especially since the similar "Omit the return keyword when not required by the language" rule is under the Patterns sections.